packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -113,16 +119,21 @@ export class EthereumAnchorService implements AnchorService {
    * @private
    */
   private _makeRequest(cidStreamPair: CidAndStream): Observable<AnchorServiceResponse> {
-    return from(
-      fetchJson(this.requestsApiEndpoint, {
-        method: 'POST',
-        body: {
-          streamId: cidStreamPair.streamId.toString(),
-          docId: cidStreamPair.streamId.toString(),
-          cid: cidStreamPair.cid.toString(),
-        },
-      })
+    return defer(() =>
+      from(
+        fetchJson(this.requestsApiEndpoint, {
+          method: 'POST',
+          body: {
+            streamId: cidStreamPair.streamId.toString(),
+            docId: cidStreamPair.streamId.toString(),
+            cid: cidStreamPair.cid.toString(),
+          },
+        })
+      )
     ).pipe(
+      retry({
```

Comment:
I just did some local testing with this PR and it looks like it swallows the error message when the request to the CAS fails.  I think it's important that it node operators still be able to tell that they're having issues connecting with the CAS, so I think this needs to log an error whenever it fails to submit a request to the CAS

Reply:
makes sense.